tuning/sft_trainer.py

```diff
@@ -129,12 +129,36 @@ def train(
         logger_name="sft_trainer_train", level=train_args.log_level
     )
 
+    resume_from_checkpoint = None
+    if train_args.output_dir:
+        os.makedirs(train_args.output_dir, exist_ok=True)
```

Comment:
Why do we need to make the output directory doesn't the trainer make it already?

Reply:
related to this failing test - https://github.com/foundation-model-stack/fms-hf-tuning/actions/runs/18341924401